src/app/core/dto/service-limits.ts

```diff
@@ -17,8 +17,10 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import {ServiceLevelType} from './service-level-type';
+
 export interface ServiceLimits {
-  serviceLevel: string,
+  serviceLevel: ServiceLevelType,
```

Comment:
I'm not really sure if this is going to be mapped correctly since string enum members are not reverse mapped in TypeScript (see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-runtime)).

I would suggest to keep `string` type in this DTO, use `ServiceLevelType` in the model, and make correct mapping in the converter using our own map (as we do for other string enums).